testpmd-operator/Makefile

```diff
@@ -66,8 +66,13 @@ deploy: kustomize
 undeploy: kustomize
 	$(KUSTOMIZE) build config/default | ${CLUSTER_CLI} delete -f -
 
+# Ensure proper digests for testpmd-container-app-mac
+ensure_digests:
+	cp roles/testpmd/defaults/main.yml.in roles/testpmd/defaults/main.yml
+	DIGEST=$$(skopeo inspect docker://quay.io/rh-nfv-int/testpmd-container-app-mac:$(TESTPMD_VER)|jq -r .Digest) && [ -n "$DIGEST" ] && sed -i -e "s/testpmd-container-app-mac@.*/testpmd-container-app-mac@$${DIGEST}\"   # $(TESTPMD_VER)/" roles/testpmd/defaults/main.yml
```

Comment:
I am not sure but I think the is needed [ -n "$${DIGEST}"] is needed otherwise, $DIGEST won't be interpreted correctly by the shell.

Reply:
Good eye here! In fact I copied this line from the same file, in the build task, so that needs to be fixed too, I'll fix that